src/libra_py/cube_file_methods.py

```diff
@@ -0,0 +1,263 @@
+#*********************************************************************************
+#*
+#* Copyright (C) 2020 Mohammad Shakiba, Brendan Smith, Alexey V. Akimov
+#* This file is distributed under the terms of the GNU General Public License
+#* as published by the Free Software Foundation, either version 2 of
+#* the License, or (at your option) any later version.
+#* See the file LICENSE in the root directory of this distribution
+#* or <http://www.gnu.org/licenses/>.
+#*
+#*********************************************************************************/
+"""
+.. module:: cube_file_methods
+   :platform: Unix, Windows
+   :synopsis: This module implements functions for processing with cube files
+.. moduleauthors:: 
+       Mohammad Shakiba, Brendan Smith, Alexey V. Akimov 
+  
+"""
+
+
+import os
+import sys
+import math
+import re
+
+if sys.platform=="cygwin":
+    from cyglibra_core import *
+elif sys.platform=="linux" or sys.platform=="linux2":
+    from liblibra_core import *
+
+import numpy as np
+from libra_py import *

```

Comment:
this is a module within libra_py, so if you import like that,  it  will become recursive - certainly a bad style
make sure you import only the modules that you need, preferably explicitly (not as import *)